src/main/java/server/router/routes/SimpleGetWithBody.java

```diff
@@ -0,0 +1,25 @@
+package server.router.routes;
+
+import java.util.List;
+import server.request.Request;
+
+public class SimpleGetWithBody extends Route {
+  public SimpleGetWithBody(Request request) {
+    super(request);
+  }
+
+  @Override
+  public String path() {
+    return "/simple_get_with_body";
+  }
+
+  @Override
+  public List<String> methods() {
+    return List.of("GET");
+  }
+
+  @Override
+  public String resource() {
```

Comment:
Resource, path or URI could mean the same in HTTP Message. Should this be just a body?

Reply:
Ok! Wasn't sure about the name for that attribute... will change to "body"